platforms/substrate/configuration/roles/create/genesis/tasks/main.yaml

```diff
@@ -0,0 +1,31 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# This tasks creates an empty listed named
+- name: Set aura_key_list to []
+  set_fact:
+    aura_key_list: []
+
+- name: Set grandpa_key_list to []
+  set_fact:
+    grandpa_key_list: []
+
+- name: Pull keys for each validator peer
+  include_tasks: nested_main.yaml
+  vars:
+    component_ns: "{{ item.name | lower }}-subs"
+    peers: "{{ item.services.peers }}"
+    vault: "{{ item.vault }}"
+  loop: "{{ network['organizations'] }}"
+
+- name: Generate genesis helm release
```

Comment:
Genesis job should be run once for the bootnode org and not in a loop on network.orgs. The next Loop should just upload the genesis file to respective org Vault